TombEngine/Objects/TR2/Entity/tr2_eagle_or_crow.cpp

```diff
@@ -16,14 +16,18 @@ namespace TEN::Entities::Creatures::TR2
 	const auto EagleBite = BiteInfo(Vector3(15.0f, 46.0f, 21.0f), 6);
 	const auto CrowBite	 = BiteInfo(Vector3(2.0f, 10.0f, 60.0f), 14);
 
-	// TODO
-	enum EagleState
+	enum EagleOrCrowState
```

Comment:
I'd rename whole `EagleOrCrow` business to a `Bird`, but it's a task for other PR, please don't do it in this one :)